### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PalkaYablyneva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ Play modes: Man vs AI, Man vs Man, AI vs AI
 
 AI supports next complexity levels: Random move, Win strategy
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:9]: wrong repo name

File: README.md [hunk 2]
```diff
@@ -8,6 +8,6 @@ Play modes: Man vs AI, Man vs Man, AI vs AI
...
 
-Actions: New, Save, Load
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:11]: wrong tag

File: README.md [hunk 3]
```diff
@@ -8,6 +8,6 @@ Play modes: Man vs AI, Man vs Man, AI vs AI
...
+Actions: New, Save, Load.
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:11]: task details missed

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PalkaYablyneva) [reply README.md:9]: wrong repo edited

File: README.md (hunk 2 above)
Comment (PalkaYablyneva) [reply README.md:11]: wrong tag edited

File: README.md (hunk 3 above)
Comment (PalkaYablyneva) [reply README.md:11]: task deteils added

File: README.md [hunk 4]
```diff
@@ -8,6 +8,6 @@ Play modes: Man vs AI, Man vs Man, AI vs AI
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:8]: wrong branches

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

**Pull request closed without merging**